frontend/models/product-list/index.ts

```diff
@@ -48,6 +49,11 @@ export async function findProductList(
       return null;
    }
    const productListImageAttributes = productList.image?.data?.attributes;
+   let deviceWiki: DeviceWiki | null = null;
+   if (productList.deviceTitle) {
+      deviceWiki = await fetchDeviceWiki(productList.deviceTitle);
+   }
```

Comment:
I don't know if it's worth it, but this eliminates a mutation:
```suggestion
   const deviceWiki = !productList.deviceTitle ? null : await fetchDeviceWiki(productList.deviceTitle);
```